Resources/doc/3-listener-support.rst

```diff
@@ -91,6 +92,17 @@ determine the request format, rather than a file extension (like foo.json).
 
 For details see :doc:`Format Listener <format_listener>`.
 
+Versioning
+----------
+
+This listener attemps to determine the current api version from different parameters of the ``Request``:
+    - the uri ``/{version}/users``
+    - a query parameter ``/users?version=v1``
+    - an ``Accept`` header ``Accept: appication/json; version=1.0``
+    - a custom header ``X-Accept-Version: v1``
```

Comment:
you shouldn't indent the list and use `*` as the indicator char. Also, please put an empty line between the paragraph and the list.